lib/integrations/hip_chat.rb

```diff
@@ -1,7 +1,43 @@
 module Integrations
   class HipChat < Base
     # NOTE: HipChat integration development still underway
-    include Integrations::MessageFormatter
+
+    def message_text
+      message = self.send(event_type.dup.concat("_message").to_sym)
+      "Your Rainforest Run (<a href=\"#{payload[:frontend_url]}\">Run ##{run[:id]}#{run_description}</a>) #{message}"
+    end
+
+    def run_description
+      run[:description].present? ? ": #{run[:description]}" : ""
```

Comment:
`run_description` is only called one and it's a one liner so I just figured it would be cleaner to move it up into `message_text`